__tests__/app.test.js

```diff
@@ -40,4 +40,41 @@ describe("GET", () => {
             })
         })
     })
+    describe('GET /api/articles/:article_id', () => {
+        test('status code: 200', () => {
```

Comment:
This doesn't need to be a separate test, we're already showing this behaviour in the test below 👍 You can just amend the one below so the description says it returns a status code of 200 as well.